src/internal/m365/collection/site/backup.go

```diff
@@ -247,11 +254,53 @@ func CollectLists(
 		return nil, false, clues.WrapWC(ctx, err, "making metadata collection")
 	}
 
-	spcs = append(spcs, col)
+	spcsMap["metadata"] = col
+
+	for _, spc := range spcsMap {
+		spcs = append(spcs, spc)
+	}
 
 	return spcs, canUsePreviousBackup, el.Failure()
 }
 
+func handleTombstones(
+	ctx context.Context,
+	bpc inject.BackupProducerConfig,
+	tombstones map[string]string,
+	spcsMap map[string]data.BackupCollection,
```

Comment:
this could just be called `collections` IMO. Using `Map` word or `Type` in general is redundant, and usually discouraged in go.

Reply:
let me make the change in the next PR in the chain